scripts/md2rst.py

```diff
@@ -27,7 +27,10 @@ def download_pandoc():
 
 
 def md2rst(text):
+    input_format = os.environ.get('FORMAT', 'markdown_github')
     download_pandoc()
-    output = pypandoc.convert_text(text, 'rst', format='md',
-     filters=[os.path.join('scripts', 'filters', 'yml_filter.py')])
```

Comment:
i'm getting lint error ` E128 continuation line under-indented for visual indent
`